tvtk/pyface/ui/qt4/scene.py

```diff
@@ -338,7 +339,7 @@ def __init__(self, parent=None, **traits):
         super(Scene, self).__init__(parent, **traits)
 
         # Setup the default picker.
-        self.picker = picker.Picker(self)
+        self.picker = picker.Picker(weakref.proxy(self))
```

Comment:
I used this idiom of passing a `weakref proxy` to make it explicit when the relationship is set.